scripts/install-multi-user.sh

```diff
@@ -56,6 +56,9 @@ readonly NIX_INSTALLED_CACERT="@cacert@"
 #readonly NIX_INSTALLED_CACERT="/nix/store/7dxhzymvy330i28ii676fl1pqwcahv2f-nss-cacert-3.49.2"
 readonly EXTRACTED_NIX_PATH="$(dirname "$0")"
 
+# allow to override identity change command
+readonly NIX_BECOME=${NIX_BECOME:-sudo}
```

Comment:
nitpick(non-blocking): The `NIX_BECOME` name is confusing to me. Something like `NIX_SUDO_PROGRAM` would convey the intent a bit better, at least to me.

Reply:
It's used in the ansible world: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_privilege_escalation.html

Reply:
At first it was `NIX_SUDO`, but then `doas` is not `sudo` - it doesn't support `sudo` config files etc. So I used Ansible way to name privilege escalation program.